packages/web-client/app/components/card-space/create-space-workflow/next-steps/index.ts

```diff
@@ -2,13 +2,23 @@ import Component from '@glimmer/component';
 import { action } from '@ember/object';
 import { inject as service } from '@ember/service';
 import RouterService from '@ember/routing/router-service';
+import { next } from '@ember/runloop';
 import { WorkflowCardComponentArgs } from '@cardstack/web-client/models/workflow';
 
 class CreateSpaceWorkflowNextStepsComponent extends Component<WorkflowCardComponentArgs> {
   @service declare router: RouterService;
 
   @action async visitSpace() {
-    // TODO
+    let username = this.args.workflowSession.getValue('username');
+
+    await this.router.transitionTo({
+      queryParams: { flow: null, 'flow-id': null },
+    });
```

Comment:
Why do we need to do this transition before the card-space.username transition?

Reply:
The workflow will still be open if we don't have this transition.

![image](https://user-images.githubusercontent.com/39579264/148529549-11272c4c-e6c1-4a1f-b1c5-a10464fe1489.png)

Card Space's creation workflow, and its trigger button, currently live in `card-space.hbs`. I think Card Space's creation workflow probably could be more cleanly separated from a single user's page (by creating a separate template and putting the creation workflow + trigger button there, in a separate PR?) so that this does not happen. Another effect of this is shown below - user space is shown next to the creation button:

![image](https://user-images.githubusercontent.com/39579264/148529689-86ba939d-f58b-4251-952d-c494d3db848b.png)

Reply:
maybe this narrow focus is _too_ narrow, since the ultimate goal would have the button lead to `username.card.space`. Which I don’t believe can be accomplished in an acceptance test. I may change it to be a link instead of a button with an action.

Reply:
but now I’m wondering if some kind of wild URL-rewriting would be helpful for this, then there wouldn’t have to be multiple deployments. I’ll check that out when I get access to Cloudflare.

Reply:
Sounds like the modal/workflow should be in the card space index route (card-space/index.hbs)